packages/react/src/components/Footer/__tests__/LegalNav.test.js

```diff
@@ -57,7 +57,7 @@ describe('<LegalNav />', () => {
 
     expect(legalNav.exists('.bx--legal-nav')).toBeTruthy();
     expect(legalNav.find('.bx--legal-nav__list-item')).toHaveLength(
-      FOOTER_NAV_DATA.length
+      FOOTER_NAV_DATA.length + 1
```

Comment:
Merely a suggestion, but we might set this `1` to a `const` variable to help remove some of its magic, and help realize why its there?

In the PR I was working on I did the following.

```javascript
const ddsPrivacyCP = 1; // we're adding an additional LI to legalNav

...

expect(legalNav.find('.bx--legal-nav__list-item')).toHaveLength(FOOTER_NAV_DATA.length + ddsPrivacyCP);
```

Reply:
Perhaps just a comment would suffice?

Reply:
Perhaps, but isn't it also used multiple times?